src/accordion/accordion.spec.ts

```diff
@@ -857,6 +872,132 @@ describe('ngb-accordion', () => {
   });
 });
 
+if (isBrowserVisible('ngb-accordion animations')) {
+  describe('ngb-accordion animations', () => {
+
+    @Component({
+      template: `
+      <ngb-accordion activeIds="first" (shown)="onShown($event)" (hidden)="onHidden($event)">
+        <ngb-panel id="first" (shown)="onFirstPanelShown()" (hidden)="onFirstPanelHidden()"></ngb-panel>
+        <ngb-panel id="second" (shown)="onSecondPanelShown()" (hidden)="onSecondPanelHidden()"></ngb-panel>
+      </ngb-accordion>
+    `,
+      host: {'[class.ngb-reduce-motion]': 'reduceMotion'}
+    })
+    class TestAnimationComponent {
+      reduceMotion = true;
+      onShown = (panelId) => panelId;
+      onHidden = (panelId) => panelId;
+      onFirstPanelShown = () => {};
```

Comment:
1. you're not using this in ANY tests, please add checks that these callbacks are actually called
2. you could have 1 function and just check what id it is called with:
```ts
onPanelShown = () = {};
```

```html
(shown)="onPanelShown('one')"
```

Reply:
Finally, I need only the check on panel one, so I'll use it without parameter.